Week-21/MySql/DataWarehouse.sql

```diff
@@ -0,0 +1,24 @@
+#Find the different warehouses in "Pune".
+SELECT DISTINCT WNAME 
+FROM WAREHOUSES 
+WHERE LOCATION = 'Pune'; 
+
+#Find a Warehouse that has the maximum number of stores.
+SELECT WID, WNAME, COUNT(*) AS StoreCount
```

Comment:
```suggestion
SELECT WID, WNAME, COUNT(*) AS storeCount
```
we use camel case while name sql vars